### PR TITLE
Update base-zh-TW.yaml

### PR DESCRIPTION
**Pull request by moyuhaken**
Adapt translation to Taiwanese grammar

### CODE REVIEW

**Comment by weasellin** [comment]
Hi @moyuhaken 

Just FYI there is another PR for similar purpose,
https://github.com/tobspr/shapez.io/pull/304
and just got merged.
In case you want to base on that change to adapt the translation,
I think you could rebase to the new master head. :)

File: translations/base-zh-TW.yaml [hunk 1]
```diff
@@ -351,25 +351,25 @@ ingame:
...
     notifications:
         newUpgrade: 有新的更新啦！
-        gameSaved: 遊戲已保存。
+        gameSaved: 遊戲已儲存。
 
     # The "Upgrades" window
     shop:
-        title: 建築升級
-        buttonUnlock: 升級
+        title: Upgrades
```
Comment (tobspr) [inline translations/base-zh-TW.yaml:358]: Any reason you removed the translation here? :)

File: translations/base-zh-TW.yaml [hunk 2]
```diff
@@ -558,64 +558,64 @@ storyRewards:
     reward_painter:
         title: 上色
         desc: >-
-            <strong>上色機</strong>已解鎖。開採一些顏色，用上色機把顏色和圖形混合，就可以為圖形著色。<br><br>備註：如果你是色盲，設置中有<strong>色盲模式</strong>可以選。
+            The <strong>painter</strong> has been unlocked - Extract some color veins (just as you do with shapes) and combine it with a shape in the painter to color them!<br><br>PS: If you are colorblind, there is a <strong>color blind mode</strong> in the settings!
```
Comment (tobspr) [inline translations/base-zh-TW.yaml:561]: And here?

File: translations/base-zh-TW.yaml [hunk 3]
```diff
@@ -725,50 +725,50 @@ settings:
...
             description: 如果您是色盲者，啟用了這設定，就可以玩遊戲了。
 
         rotationByBuilding:
-            title: 依建築類型旋轉
+            title: Rotation by building type
             description: >-
-                每個建築類型，將會分別記住您最後一次使用的旋轉方向。
-                如果您常常切換不同類型的建築，這樣可能會更方便。
-
+                Each building type remembers the rotation you last set it to individually.
```
Comment (tobspr) [inline translations/base-zh-TW.yaml:765]: And here? :)

File: translations/base-zh-TW.yaml [hunk 4]
```diff
@@ -189,7 +189,7 @@ mainMenu:
 dialogs:
     buttons:
         ok: 確認
-        delete: 刪除
+        delete: 删除
```
Comment (weasellin) [inline translations/base-zh-TW.yaml:192]: Any reason you revert the translation from trad. Ch. to simp. Ch. here? :)

**Comment by tobspr** [comment]
Closing this due to no response - feel free to reopen once the changes are done! :)

**Pull request closed without merging**